CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -1078,6 +1151,13 @@ public CommandBase Delete
                             {
                                 throw new NotImplementedException("Not implemented");
                             }
+                            else if (itemType == typeof(CertificateModel))
+                            {
+                                CertificateModel certificate = item as CertificateModel;
+
+                                objectType = "Certificate";
+                                objectName = certificate.Thumbprint;
```

Comment:
For UI purposes the thumbprint alone is visually clearer - there is no ambiguity at the moment, and even if we later add non-SHA1 thumbprints, there is extremely unlikely to be any ambiguity.